lesson2/MainActivity.java/lib/src/main/java/com/example/Account.java

```diff
@@ -3,19 +3,6 @@
 /**
  * Created by erica on 9/8/16.
  */
-//public class Account {
-//
-//    private long amount;
-//
-//    public static long getAmount(long amount) {
-//        return amount;
-//    }
-//
-//    public static long setAmount(long new_amount) {
-//        long amount = new_amount;
-//        return amount;
-//    }
-
```

Comment:
Generally try to delete commented-out lines that you didn't use in your final app before submitting your homework